QuizApp/QuizApp/UI/QuizDetails/QuizDetailsViewController.swift

```diff
@@ -0,0 +1,124 @@
+import Combine
+import UIKit
+
+class QuizDetailsViewController: UIViewController {
+
+    private let gradient = CAGradientLayer()
+
+    private var quiz: Quiz!
```

Comment:
Do not store quiz to VC. VC should always read values from VM. So, store Quiz to VM.